wcomponents-core/src/test/java/com/github/bordertech/wcomponents/WMultiFileWidget_Test.java

```diff
@@ -309,6 +309,18 @@ public void testMaxFileSizeAccessors() {
 				(long) 2);
 	}
 
+	@Test
+	public void testDuplicateComponentModels() {
+		WMultiFileWidget multiFileWidget = new WMultiFileWidget();
+		assertNoDuplicateComponentModels(multiFileWidget,"maxFileSize", 2012312312);
+		assertNoDuplicateComponentModels(multiFileWidget, "maxFiles", 123);
+		assertNoDuplicateComponentModels(multiFileWidget, "newUpload", true);
+		assertNoDuplicateComponentModels(multiFileWidget, "useThumbnails", true);
+		assertNoDuplicateComponentModels(multiFileWidget, "thumbnailPosition", WLink.ImagePosition.SOUTH);
+		assertNoDuplicateComponentModels(multiFileWidget, "thumbnailSize", new Dimension(22,33));
+//		assertNoDuplicateComponentModels(multiFileWidget, "fileUploadRequestId", "testId"); // No such method exception as it's a private method
```

Comment:
@jonathanaustin Setter methods that have an access modifier other than public can't be found by `invokeSetMethod`. When making these methods `public `and uncommenting lines such as this one the tests pass. Is that good enough, or would you rather me write special tests that replicate all the tests in `assertNoDuplicateModels`, place them in this test file, and make the methods `protected`/no modifier?

Reply:
@Jonno1809 At the moment the test class can only access public methods. To fix this is beyond the scope of this PR (even though it needs fixing). Raise a new Issue about the ability for the test harness to access "non-public" methods.
So...for this PR leave it commented out.